### ISSUE TITLE
servicec times - could not parse line: Service Time Percentiles

### ISSUE THREAD

**Issue by roffe** [boynux/squid-exporter#42]
When i fire up the exporter and try to scrape i get the following in the logs:

```
2020/08/11 10:14:59 servicec times - could not parse line: Service Time Percentiles            5 min    60 min:
```

output for `squidclient mgr:info`

```
Squid Object Cache: Version 4.10
Build Info: Ubuntu linux
Service Name: squid
Start Time:	Mon, 27 Jul 2020 12:47:38 GMT
Current Time:	Tue, 11 Aug 2020 10:16:13 GMT
Connection information for squid:
	Number of clients accessing cache:	4
	Number of HTTP requests received:	3126
	Number of ICP messages received:	0
	Number of ICP messages sent:	0
	Number of queued ICP replies:	0
	Number of HTCP messages received:	0
	Number of HTCP messages sent:	0
	Request failure ratio:	 0.00
	Average HTTP requests per minute since start:	0.1
	Average ICP messages per minute since start:	0.0
	Select loop called: 3595989 times, 357.875 ms avg
Cache information for squid:
	Hits as % of all requests:	5min: 0.0%, 60min: 0.0%
	Hits as % of bytes sent:	5min: 78.3%, 60min: 29.3%
	Memory hits as % of hit requests:	5min: 0.0%, 60min: 0.0%
	Disk hits as % of hit requests:	5min: 0.0%, 60min: 0.0%
	Storage Swap size:	0 KB
	Storage Swap capacity:	 0.0% used,  0.0% free
	Storage Mem size:	216 KB
	Storage Mem capacity:	 0.1% used, 99.9% free
	Mean Object Size:	0.00 KB
	Requests given to unlinkd:	0
Median Service Times (seconds)  5 min    60 min:
	HTTP Requests (All):   0.00000  0.00000
	Cache Misses:          0.00000  0.00000
	Cache Hits:            0.00000  0.00000
	Near Hits:             0.00000  0.00000
	Not-Modified Replies:  0.00000  0.00000
	DNS Lookups:           0.00000  0.00394
	ICP Queries:           0.00000  0.00000
Resource usage for squid:
	UP Time:	1286914.932 seconds
	CPU Time:	102.564 seconds
	CPU Usage:	0.01%
	CPU Usage, 5 minute avg:	0.02%
	CPU Usage, 60 minute avg:	0.02%
	Maximum Resident Size: 118800 KB
	Page faults with physical i/o: 5
Memory accounted for:
	Total accounted:         2603 KB
	memPoolAlloc calls:   3129793
	memPoolFree calls:    3133735
File descriptor usage for squid:
	Maximum number of file descriptors:   1024
	Largest file desc currently in use:     26
	Number of file desc currently in use:   18
	Files queued for open:                   0
	Available number of file descriptors: 1006
	Reserved number of file descriptors:   100
	Store Disk files open:                   0
Internal Data Structures:
	    53 StoreEntries
	    53 StoreEntries with MemObjects
	     1 Hot Object Cache Items
	     0 on-disk objects
```

This squid instance is used as a forwarding proxy i want to monitor, so it has no backends if that might be the culprit

**Comment by roffe** [comment]
https://github.com/boynux/squid-exporter/pull/43 might be a part to the solution

**Comment by boynux** [comment]
Hi @roffe, thanks for reporting this issue. Could you check v1.8.3 version and see if that works with your Squid instance?
https://github.com/boynux/squid-exporter/releases/tag/1.8.3

**Comment by boynux** [comment]
Fixed in #43
